sale_invoice_plan/models/sale.py

```diff
@@ -0,0 +1,215 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+from dateutil.relativedelta import relativedelta
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError, ValidationError
+from odoo.addons import decimal_precision as dp
+from odoo.tools.float_utils import float_compare, float_round as round
```

Comment:
Rename round variable to something else as this is a built-in method of python.

Travis: sale_invoice_plan/models/sale.py:7: [W0622(redefined-builtin), ] Redefining built-in 'round'